graph/src/data/query/error.rs

```diff
@@ -224,7 +225,8 @@ impl fmt::Display for QueryExecutionError {
             Panic(msg) => write!(f, "panic processing query: {}", msg),
             EventStreamError => write!(f, "error in the subscription event stream"),
             FulltextQueryRequiresFilter => write!(f, "fulltext search queries can only use EntityFilter::Equal"),
-            TooExpensive => write!(f, "query is too expensive")
+            TooExpensive => write!(f, "query is too expensive"),
+            DeploymentReverted => write!(f, "the subgraph deployment was reverted while executing the query"),
```

Comment:
This error message isn't very clear, to me it almost sounds as if the deployment was entirely erased. Maybe "The chain had a reorganization while the query was executing".

Reply:
I changed it to `the chain was reorganized while executing the query`. What do you think?

Reply:
Better, thanks